implementations/index.md

```diff
@@ -188,6 +190,10 @@ includes related resources.
 
 ### <a href="#server-libraries-java" id="server-libraries-java" class="headerlink"></a> Java
 
+
+* [crnk.io](https://www.crnk.io) is a JSON API framework for clients and servers. On the server-side it comes with a rich set
+  of integrations (Servlet, JAX-RS, Spring, JPA, Bean Validation, Zipkin and more), bulk updates with JSON Patch, client stub 
+  generation for TypeScript and a module API for third-party contributions.
```

Comment:
Could you please move this to the bottom of the list? We try to keep the lists chronological.